corgi/core/models.py

```diff
@@ -513,7 +521,78 @@ def get_ofuri(self) -> str:
     @property
     def manifest(self) -> str:
         """Return an SPDX-style manifest in JSON format."""
-        return ProductManifestFile(self).render_content()
+        doc = Document(spdx_id="SPDXRef-DOCUMENT", version=Version(2, 2), name=self.name)
+        doc.namespace = f"https://access.redhat.com/security/data/manifest/spdx/{self.name}"
+        doc.data_license = License.from_identifier("CC0-1.0")
+        doc.creation_info.add_creator(
+            Organization("Red Hat Product Security", "secalert@redhat.com")
+        )
+        doc.creation_info.set_created_now()
+        for component in self.get_latest_components():
+            comp_spdx_id = self.add_manifest_package(component, doc)
```

Comment:
You can make this a Django template tag, or just a Component model property that outputs the SPDX JSON that you want to use:
```
{% for component in product.get_latest_components %}
    {{component.get_manifest_json}},
{% endfor %}
```
Then there's no need to switch to a library just for this one usecase (although switching to a library may be a good idea for other reasons).